tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationControllerBuilder.cs

```diff
@@ -48,15 +53,32 @@ internal OrganisationControllerBuilder WithUpdateOrganisation(Organisation organ
             return this;
         }
 
+        internal OrganisationControllerBuilder WithCreateOrganisation()
+        {
+            var repositoryMock = new Mock<IOrganisationRepository>();
+            repositoryMock.Setup(x => x.CreateOrganisationAsync(It.IsAny<Organisation>()));
+            _organisationRepository = repositoryMock.Object;
+            return this;
+        }
+
         public OrganisationControllerBuilder WithOrganisationRepository(IOrganisationRepository organisationRepository)
         {
             _organisationRepository = organisationRepository;
             return this;
         }
 
+        public OrganisationControllerBuilder WithCreateOrganisationService(bool isSuccess, string resultValue)
```

Comment:
I think this could probably be named better... Perhaps `WithCreateOrganisationResult` or something along those lines?

Reply:
`WithCreateOrganisationResult` sounds like an improvement.
How about `WithCreateOrganisationReturningResult`

Reply:
`WithCreateOrganisationReturningResult` sounds good to me.

Reply:
👍

Reply:
On second thought, wouldn't `WithCreateOrganisationServiceReturningResult` be even better? Given that we're setting up the _service_ and not the organisation to return a result.